ru/mail/branding/nightly/brand.dtd

```diff
@@ -1,3 +1,3 @@
-<!ENTITY  brandShortName        "Postbox">
+﻿<!ENTITY  brandShortName        "Postbox">
 <!ENTITY  brandFullName         "Postbox">
 <!ENTITY  vendorShortName       "">
```

Comment:
"Brand" stands for "Postbox" software product here, while "Vendor" is "Postbox, Inc", i. e. the company that developed Postbox 5. Please, correct me if I'm wrong.